src/TimePicker/TimePicker.js

```diff
@@ -162,7 +162,18 @@ var MeridiemPicker = kind(
 	
 	ariaObservers: [
 		{path: 'value', method: function () {
-			this.setAriaAttribute('aria-valuetext', this.meridiems[this.value]);
+			// When value is changed, it reads only value
+			if (this.spotted) {
+				this.set('accessibilityHint', null);
+				this.setAriaAttribute('aria-valuetext', this.meridiems[this.value].name);
+			}
+		}},
+		{path: 'spotted',  method: function () {
+			// When spotlight is focused, it reads value with hint
+			if (this.spotted) {
+				this.set('accessibilityHint', $L('change a value with up/down button'));
```

Comment:
Slash of up/down could cause a problem for TTS. Ask UI to change string to up and down